packages/statemanager/src/stateManager.ts

```diff
@@ -31,12 +31,12 @@ import {
 import debugDefault from 'debug'
 import { keccak256 } from 'ethereum-cryptography/keccak.js'
 
-import { CacheType, OriginalStorageCache } from './cache/index.js'
+import { OriginalStorageCache } from './cache/index.js'
 import { modifyAccountFields } from './util.js'
 
-import { CODEHASH_PREFIX, Caches, type DefaultStateManagerOpts } from './index.js'
```

Comment:
👀 

Ah, ok, now I get it.

So seems my observation was correct (?).

Cool, great dive and find! 🎉 🙂